java/ql/src/semmle/code/java/Statement.qll

```diff
@@ -733,12 +735,12 @@ class LabeledStmt extends Stmt, @labeledstmt {
   /** Gets the label of this labeled statement. */
   string getLabel() { namestrings(result, _, this) }
 
-  /** Gets a printable representation of this statement. May include more detail than `toString()`. */
   override string pp() { result = this.getLabel() + ": " + this.getStmt().pp() }
 
-  /** This statement's Halstead ID (used to compute Halstead metrics). */
   override string getHalsteadID() { result = this.getLabel() + ":" }
 
+  override string toString() { result = "Label:" }
```

Comment:
Might cause confusion when there is actually a label called `Label`, but is at least more useful than `...:` or similar.